ocaml/xapi/api_server.ml

```diff
@@ -250,24 +250,27 @@ let is_host_is_slave_error (response : Rpc.response) =
       false
 
 let create_thumbprint_header req response =
-  let include_thumbprint =
+  let hash_type_option =
```

Comment:
There's no need to add option to the name, it's obvious when treating the result that it's an option
```suggestion
  let hash_type =
```